### ISSUE TITLE
Array order

### ISSUE THREAD

**Issue by mbrandonw** [jpsim/Yams#121]
Hey, great library!

I'm using it currently to read in a YML file of an array of objects, sort the objects into a normalized order, and then write the YML back to the file. However, the output encoded YML does not have the same order as the array I have in memory.

Is that expected?

Thanks!

**Comment by jpsim** [comment]
No, that’s not expected. Can you share repro steps so we can look into this?

**Comment by jpsim** [comment]
No problem. Happy to see you using Yams! If you're using this in something OSS, please share 😄
